src/components/ui/Select/Select.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { Select as UISelect } from '@faststore/ui'
+import type { SelectProps } from '@faststore/ui'
+import { CaretDown as CaretDownIcon } from 'phosphor-react'
+
+import './select.scss'
+
+type SelectOptions = {
+  [key: string]: string
+}
+
+interface UISelectProps extends SelectProps {
+  /*
+   * Redefines the id property to be required when using the Select component. The
+   * id will be used to link the UISelect component and its label.
+   */
+  id: string
+  /*
+   * Defines the CSS class string that will be forwarded to the wrapping div "className" prop.
+   */
+  classes?: string
```

Comment:
Why not use the same pattern that react uses, className? By doing this, is transparent for the Select user?

Reply:
I spoke with other teammates and we're sticking to this `classes` naming pattern to indicate that this is not the "native" `className` react attribute, but instead a new attribute that comes from the outside and will be forwarded to some react node internal to that component.

Reply:
Update: Apparently that's only when the interface defined does not extend an interface that already has `className` implemented. You're right, my bad.